src/components/EditorSideBar/index.tsx

```diff
@@ -0,0 +1,17 @@
+import EditorSideBarItem from "./EditorSideBarItem";
+import sideBarItemList from "./const";
+import { Box, Flex } from "@chakra-ui/react";
+
+const EditorSideBar = () => {
+  return (
+    <Box bg="background.primary" w="98px" h="calc(100vh)" position="fixed" top="71px" left="0">
```

Comment:
Please do some research about how to use the calc CSS funtion. It was meant to be a calculation. Because you know the size of the navbar, you know the screen size(which is vh100). Based on these 2 you can calculate the height of sidebar.